packages/api/controllers/nomination.js

```diff
@@ -14,10 +15,30 @@ const getNominationById = async (req, res) => {
     if (nomination) {
       return res.status(200).json({ nomination });
     }
-    return res.status(404).send('Nomination with the specified ID does not exist!');
+    return res
+      .status(404)
+      .send('Nomination with the specified ID does not exist!');
   } catch (error) {
     return res.status(500).send(error.message);
   }
 };
 
-module.exports = { getNominationById };
+const createNomination = async (req, res) => {
+  if (req.body === null) {
+    return res.status(404).send('Required fields are not completed');
+  }
+  try {
+    const nomination = await db.Nomination.create(req.body);
+
+    return res.status(201).json({ nomination });
+  } catch (error) {
+    if (error instanceof ValidationError) {
+      console.log('400 validation error', error);
+      return res.status(400).json({ error: error.message });
+    }
+
+    return res.status(500).json({ error: error.message });
```

Comment:
We may want to `console.error` the message here, too, so we can debug when things go wrong.